util/dashboard/gen_dashboard_entry.py

```diff
@@ -81,7 +82,7 @@ def gen_dashboard_html(hjson_path, outfile):
                     "        <td>&nbsp;</td>\n")
     if 'notes' in obj:
         genout(outfile,
-                    "        <td>" + html.escape(obj['notes']) + "</td>\n")
+                    "        <td>" + mk.markdown(obj['notes']) + "</td>\n")
```

Comment:
@sjgitty I've revised the dashboard to parse markdown format in `notes` field.

Reply:
I think you should return markdown here and then switch to `{{% %}}`-style partials which will render markdown from the content they produce (AIUI).

Reply:
One thing to note, the table doesn't support multi-line cell as far as I know. Can Blackfriday (hugo) use alternate table format?